benchmarks/basic_benchmarks.py

```diff
@@ -2,6 +2,7 @@
 # See "Writing benchmarks" in the asv docs for more information.
 
 import liionpack as lp
+import pybamm
 
 
 class BasicBenchmark:
```

Comment:
This test for some reason doesn't work. Is the syntax correct?

Reply:
It runs locally so I guess it times out. I will shorten it

Reply:
I have reduced the time and added timeout